### DISCUSSION TITLE
App is worthless

### DISCUSSION THREAD

**Discussion by TheRedJalapeno** [community/community#133350] [Mobile]
There is something wrong with my GitHub Pages configuration. So to solve this I try to log in via Mobile. Mobile requires I use the app to authenticate and log in. Okay done. The app does not seem to have the same functionality as the website, which is normal, and the whole reason apps as a category are stupid. But that's okay because I have the website. Now I am on mobile on web. I click on my repo so I can go to settings and Pages, but lo, clicking on the repo sends me back to the app. The app is not functional. It doesn't do anything for me. But I have to have the app for authentication to log in. Am I supposed to uninstall the app every time I log in so I can do what I need to do on the website? Can you please just delete your stupid app so I can use your website? Oh and now I can't even submit an issue on the app, see screenshot.

![Screenshot_20240724-010952.png](https://github.com/user-attachments/assets/b6161c97-f64a-49cc-b4a4-d0346b934661)

**Comment by stevepopovich** [comment]
Hi @RobertDAllen sorry you are having a bad experience. Hopefully I can help mitigate some of this stress. 

Our app does not have any Pages settings, so yes, you will have to use the website for that. 

Your browser should have a setting about "Open links in app" or similar. You can switch that setting to `Ask` or `Never` so the links don't go to the app. That should help the link issue a bit.

Finally, about the support issue. I am having trouble reproducing that problem. Is this consistent for you?

Let me know if any of this helps. Sorry for the frustration.

**Reply by stevepopovich** [reply]
Actually, remembering some more solutions here if you are still having issues:

1. You can always use a fallback method for 2FA that isn't mobile. Or configure your 2FA to use a different method: https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication
2. You can make the GitHub app itself not accept any deeplinks, if you want: 
- Long press on the app -> Go to the `i` -> Scroll down to `Open by default` ->  Turn off `Open unsupported links`.

This will preserve your other apps deeplinks from your browser and block GitHub from taking deeplinks in other apps that are not your browser. Note turning this off can affect your ability to login to the GitHub app.
